modules/redis/collector.go

```diff
@@ -0,0 +1,76 @@
+package redis
+
+import (
+	"github.com/NETWAYS/support-collector/internal/collection"
+	"github.com/NETWAYS/support-collector/internal/obfuscate"
+	"github.com/NETWAYS/support-collector/internal/util"
+	"path/filepath"
+)
+
+const ModuleName = "redis"
+
+var relevantPaths = []string{
+	"/etc/redis",
+	"/etc/redis.conf",
+}
+
+var files = []string{
+	"/etc/redis*",
+}
+
+var optionalFiles = []string{
+	"/etc/logrotate.d/redis*",
+}
+
+var detailedFiles = []string{
+	"/var/log/redis/redis-server.log",
+	"/var/log/redis/redis.log",
+}
+
+var possibleDaemons = []string{
+	"/lib/systemd/system/redis-server.service",
+	"/lib/systemd/system/redis-server@.service",
```

Comment:
What about the `icingadb-redis`?

Reply:
This module is only for redis itself.  
icingadb-redis is part of the module `icingadb`

Reply:
check